src/audio/google_rtc_audio_processing_mock.c

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2021 Google LLC.
+//
+// Author: Lionel Koenig <lionelk@google.com>
```

Comment:
would it be possible to make this a part of the SOF cmocka test suite? Any reason not to do that?

Reply:
This mock is a vlaid component which just does not do the processing. It is used to verify that the firmware compiles and to be able to run code even if one has no access to the third party library.

Reply:
This is similar to the smart amp test, its an in place component for those who do not have the lib.